qute.ls/com.redhat.qute.ls/src/main/java/com/redhat/qute/project/tags/UserTagRegistry.java

```diff
@@ -137,8 +142,9 @@ protected CompletableFuture<List<UserTagInfo>> getBinaryUserTags(QuteUserTagPara
 	 * 
 	 * @return the src/main/resources/templates/tags directory.
 	 */
-	public Path getTagsDir() {
-		return tagsDir;
+	public Path getPreferredTagsDir() {
+		// TODO : select the non null tags dir

```

Comment:
Should this TODO: be addressed in this PR?

Reply:
You are right, fixed